terraform/modules/public_albs/auth_listener.tf

```diff
@@ -1,4 +1,4 @@
-#The auth alb is shared by keycloak and the auth service so we define the listener here and the rules in each repository
+#The auth alb was shared by keycloak and the auth service so we defined the listener here and the rules in each repository
```

Comment:
Should comments still reference things we don't do anymore even if its past tense?